app/model/sdl/NonMediaModel.js

```diff
@@ -273,6 +273,33 @@ SDL.SDLNonMediaModel = SDL.ABSAppModel.extend({
   sdlSetMediaClockTimer: function() {
 
     return;
-  }
+  },
+
+  classNameBindings: [
+    'dayMode',
+    'nightMode',
+    'highLightedMode'
+  ],
+
+  dayMode:false,
+  nightMode:false,
+  highLightedMode:false,
+
+    /**
+     * @description Method applies image mode view for model.
+     * @param {Number}
+     */
+  setMode:function(mode){
+    if(this.isTemplate){
+      this.set('dayMode',false);
+      this.set('nightMode',false);
+      this.set('highLightedMode',false);
```

Comment:
@YarikMamykin you can remove these three lines as the next three lines overwrites them anyway

Reply:
@AKalinich-Luxoft removed in https://github.com/LuxoftSDL/sdl_hmi/pull/11/commits/0f765eb595655bb7f015bf8bd62019c4e6961d91